lib/workers/repository/init/merge.ts

```diff
@@ -51,7 +51,12 @@ export async function detectRepoFileConfig(): Promise<RepoFileConfig> {
   const cache = getCache();
   let { configFileName } = cache;
   if (configFileName) {
-    const configFileRaw = await platform.getRawFile(configFileName);
+    let configFileRaw: string | null;
+    try {
+      configFileRaw = await platform.getRawFile(configFileName);
+    } catch (err) {
+      configFileRaw = null;
```

Comment:
i think we should at least log the message for trouble shouting

Reply:
On line 69 we handle the logging for undefined/null configFileRaw.

Reply:
but we should at least throw external host errors